QGCApplication.pro

```diff
@@ -78,6 +81,11 @@ QT += testlib
 # OS Specific settings
 #
 
+AndroidBuild {
+    DEFINES += __android__
+    DEFINES += __STDC_LIMIT_MACROS
```

Comment:
Got it. This was a merge issue for when there was only one project file. I will move it.